mcstatus/protocol/connection.py

```diff
@@ -337,7 +339,7 @@ def __repr__(self) -> str:
     @staticmethod
     def _unpack(format_: str, data: bytes) -> int:
         """Unpack data as bytes with format in big-endian."""
-        return struct.unpack(">" + format_, bytes(data))[0]
+        return cast(int, struct.unpack(">" + format_, bytes(data))[0])
```

Comment:
This should be reverted, the type cast is not necessary in here, as `struct.unpack` returns `Any`, so there shouldn't be any issues with the return type-hint being `int`. This can be left as it was.

Is mypy reporting an issue here? We're using pyright in the codebase, which seems to be fine with this, which is why I'm confused as to why you even made this change here.

Reply:
Yes, I have mypy set to use `no-return-any`, I can revert this.

Reply:
Reverted in [64f7e0f](https://github.com/py-mine/mcstatus/pull/555/commits/64f7e0f49a5a95e1f346665510bdb18ca3f50d70)

Reply:
You should use pyright as we use it for this project.